spring-vault-core/src/main/java/org/springframework/vault/core/VaultPkiOperations.java

```diff
@@ -108,4 +109,31 @@ enum Encoding {
 
 	}
 
+	/**
+	 * Retrieves the specified issuer's certificate. Includes the full ca_chain of the
+	 * issuer.
+	 * @param issuer reference to an existing issuer, either by Vault-generated
+	 * identifier, or the name assigned to an issuer. If empty or {@literal null}, the
+	 * issuer will default to the literal string default to refer to the currently
+	 * configured default issuer.
+	 * @return the {@link VaultIssuerCertificateRequestResponse} containing a
+	 * {@link org.springframework.vault.support.Certificate}
+	 * @see <a href=
+	 * "https://www.vaultproject.io/api/secret/pki/#read-issuer-certificate">GET *
+	 * /pki/issuer/:issuer_ref/json</a>
+	 *
+	 */
+	VaultIssuerCertificateRequestResponse getIssuerCertificate(@Nullable String issuer) throws VaultException;
```

Comment:
I'm not sure that this argument being `@Nullable` is a good design. I understand the background, however, the defaulting to the default issuer could be handled directly by the caller, effectively us reducing the complexity.

Reply:
I was a bit in doubt and I agree better to move this to the caller. Adjusted the code accordingly